src/pacemaker.c

```diff
@@ -72,6 +72,11 @@ static int pcmk_write_ticket_atomic(struct ticket_config *tk, int grant)
 			(int64_t)wall_ts(&tk->term_expires),
 			(int64_t)tk->current_term);
 
+	if (rv < 0 || rv >= COMMAND_MAX) {
+		log_error("pcmk_write_ticket_atomic: crm_ticket cmd too long");
+		return -1;
```

Comment:
Yeah ... I guess negative return value will happen mostly for different reasons. On the other hand I believe if one of these snprintfs fails, it will be because of too long string. So I've changed the error message to `cannot format crm_ticket cmdline (probably too long)`. What do you think?

Reply:
I think that is fine.